src/nodeComponents/click/click.js

```diff
@@ -25,29 +25,36 @@ export default (node) =>
    * .find('Cube_1')
    * .click(true); // you can also pass in `debug` as `true` to see the cursor
    */
-    (debug = false) => {
-    const position = node.getWorldPosition(new Vector3());
+  (debug = false) => {
+    // Find the centre point of the 3D node
+    node.updateMatrixWorld();
+    const box = new THREE.Box3().setFromObject(node);
+    const position = box.getCenter(new THREE.Vector3());
+    position.applyMatrix4(node.matrixWorld);
     const canvas = store.get(C.RENDERER).domElement;
 
-    const screenSpace = worldToScreenSpace(
-      position.x,
-      position.y,
-      position.z,
-      canvas.clientWidth,
-      canvas.clientHeight
-    );
+    const screenSpace = worldToScreenSpace(position.x, position.y, position.z, canvas.clientWidth, canvas.clientHeight);
```

Comment:
For consistency, can we have the parameters on a multi-line? Some editors are a PITA with lines that go beyond a certain count.